### DISCUSSION TITLE
Telemetry

### DISCUSSION THREAD

**Discussion by Billy60** [FrSkyRC/ETHOS-Feedback-Community#4724] [General]
Would it be possible to have a telemetric field which would stick to a maximum value ? For instance, 'ALT+ ' would keep in memory the highest altitude the plane reached. Then, back on the ground it would be easy to read it and not to have to look for it during flight.

Thanks!

**Comment by Dsrenger** [comment]
Already available

![image](https://github.com/user-attachments/assets/eb2cfd28-8097-42f9-a996-a944ae24d04a)

Longpress on "Flughöhe" (which means height in german ;))....you can always longpress on a field if it hast the 3 lines in the corner.

Choose "max"
![image](https://github.com/user-attachments/assets/7f59e428-ebdb-40ba-b184-5125a865ed67)

You can doo that with nearly all telemetry values. usually it offers at least min and max

**Comment by Billy60** [comment]
Thanks for the info !!
